website/docs/r/iam_server_certificate.html.markdown

```diff
@@ -117,6 +117,7 @@ In addition to all arguments above, the following attributes are exported:
 * `id` - The unique Server Certificate name
 * `name` - The name of the Server Certificate
 * `arn` - The Amazon Resource Name (ARN) specifying the server certificate.
+* `tags` - Key-value mapping of tags for the server certificate.
```

Comment:
Add documentation for new `expiration` and `upload_date` attributes.

Reply:
Added

Reply:
Tags should be documented in the arguments section -- will move up on merge.